LINK/root/.ansible.cfg

```diff
@@ -1,3 +1,3 @@
 [defaults]
 stdout_callback = json
-roles_path = /etc/ansible/roles:/var/www/miq/vmdb/content/ansible_consolidated/roles
+roles_path = ~/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/var/www/miq/vmdb/content/ansible_consolidated/roles
```

Comment:
This is the ansible config for the root user so can you change the first path to be absolute? `/root/.ansible/roles`?

Reply:
Sure. Done.